src/components/SearchInput.vue

```diff
@@ -3,7 +3,7 @@
     <template v-if="label">
       <p-label for="search" :label="label" class="search__label" />
     </template>
-    <p-text-input v-model="internalValue" type="search" :placeholder="placeholder" class="search__input">
+    <p-text-input v-model="internalValue" type="search" :small="small" :placeholder="placeholder" class="search__input">
```

Comment:
This is where you can use the new shorthand! Not a real suggestion, just an example
```suggestion
    <p-text-input v-model="internalValue" type="search" :small :placeholder class="search__input">
```

Reply:
Ah yeah, that'll require a small shim for `small` in the meantime but I'm ok with that.

Reply:
not saying you should or have to use it, just pointing it out since you noticed it in my PR yesterday

Reply:
Ah sorry that comment got added to the wrong thread - love the suggestion, applying it!